### PR TITLE
Bugfix: Supercritical fix for Pype

### PR DESCRIPTION
**Pull request by antirotor**
# ❗ ❗ ❗ WARNING: CRITICAL BUG DETECTED
## Merge NOW!

Pype 3 is lacking critical feature. Without it it can destroy world or delete all porn. This PR is fixing it.

### CODE REVIEW

**Comment by iLLiCiTiT** [comment]
I agree :D That gave me new motivation to continue with Pype.

File: pype/lib/terminal_splash.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    animation_length = int(len(animation) / frame_size)
+    current_frame = 0
+    for _ in range(animation_length):
+        if sys.platform.startswith('win'):
+            os.system('cls')
+        else:
+            os.system('clear')
+        for scanline in animation[current_frame:current_frame + frame_size]:
+            print(scanline.rstrip())
+        sleep(0.05)
```
Comment (mkolar) [inline pype/lib/terminal_splash.py:24]: For everyone's sake, let's make this 0.02 shall we?

**Review by mkolar — CHANGES_REQUESTED** [review]
facepalm-font-friday

**Comment by antirotor** [comment]
Big yet unfamiliar
Think bigger than the biggest bar
Think big, that's only half as large
Bigger, better, twice as hard
Outsized, extra large
Velocity very fast
Bigger than the closest star
Bigger than the motor car

Quick, quiet, confident
Comfortable, permanent
Undisputed every tense
Not a trace of what went left
More equal than the best
Unparalleled success
Everybody
Be impressed!

**Review by mkolar — APPROVED** [review]

**Pull request merged**